src/app/search/components/patient_search/patient_search.directive.js

```diff
@@ -22,16 +22,20 @@
         function PatientSearchController($log, commonService) {
             var vm = this;
 
-            vm.assembledDob = assembledDob;
             vm.errorCount = errorCount;
             vm.searchForPatient = searchForPatient;
+            vm.assembledDob = assembledDob; 
 
             activate();
 
             ////////////////////////////////////////////////////////////////////
 
             function activate () {
                 vm.query = {};
+                vm.query.patientNames = [{givenName: [''], nameType: { code: 'L', description: 'Legal Name'} }];
+                vm.query.dob = {};
+                vm.query.dob.month = "";
+                vm.query.dob.day = "";
```

Comment:
Not entirely sure why, but everywhere else uses single quotes `'` as opposed to double quotes `"`